tests/bluetooth/host/buf/bt_buf_get_evt/src/main.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/zephyr.h>
+
+void test_main(void)
+{
+	uint32_t state;
```

Comment:
Considering that the state is theoretically a parameter to the run call and could be used for future uses, it's probably a good idea to initialize the value so that a random value of the stack is not misinterpreted as an initialization value. This is less relevant if removed due to deprecation (see subsequent comment).